pkg/cmd/install.go

```diff
@@ -375,15 +375,14 @@ func (o *installCmdOptions) installOperator(cmd *cobra.Command, output *kubernet
 		}
 	}
 
-	strategy := ""
+	message := "Camel K installed in namespace " + namespace
```

Comment:
It's a scratch branch - this will disappear with the next rebase.

PS: by convention I use '--...' for commits that will later be squashed before the real commit